examples/defaulter-gen/Makefile

```diff
@@ -2,7 +2,7 @@ TOOL=defaulter-gen
 
 .PHONY : gen test verify git_verify_clean git_verify_no_change
 test:
-	go test ./...
+	go test ./_output_tests/marker/...
```

Comment:
This isn't ideal but none of the other output_tests packages compile without adding the deepcopy and conversion generated code. Given that this PR is pretty big already, I think it's best to do that in a follow up.